lichess-bot.py

```diff
@@ -113,10 +120,11 @@ def start(li, user_profile, engine_factory, config):
 
 @backoff.on_exception(backoff.expo, BaseException, max_time=600, giveup=is_final)
 def play_game(li, game_id, control_queue, engine_factory, user_profile, config):
-    updates = li.get_game_stream(game_id).iter_lines()
+    response = li.get_game_stream(game_id)
+    lines = response.iter_lines()
 
     #Initial response of stream will be the full game info. Store it
-    game = model.Game(json.loads(next(updates).decode('utf-8')), user_profile["username"], li.baseUrl, config.get("abort_time", 20))
+    game = model.Game(json.loads(next(lines).decode('utf-8')), user_profile["username"], li.baseUrl, config.get("abort_time", 20))
```

Comment:
You can remove the `.decode('utf-8')` here since we do it earlier.

Reply:
OK, I'll remove this.